applications/Chat/evaluate/README.md

```diff
@@ -24,58 +23,232 @@ The whole evaluation pipeline consists of two methods:
 
 Our evaluation pipeline examines the model's capability using 10 categories of questions. The following table introduces each category:
 
-| Evaluation Category | <center>Description</center>                                                  |
```

Comment:
Please use markdown instead of html. html is hard to read for human.

Reply:
Ok.